web/server/static/static-threaded.c

```diff
@@ -482,6 +496,11 @@ void *socket_listen_main_static_threaded(void *ptr) {
 
             static_workers_private_data = callocz((size_t)static_threaded_workers_count, sizeof(struct web_server_static_threaded_worker));
 
+            if (is_there_react_dashboard()) {
```

Comment:
Is there a way to have non-static-threaded web server? In theory we should allow dynamic but I can't find this codepath for some reason.

Reply:
Considering what I saw in parts commented on the web server, I think it was in the past, but it was moved to be static, so it is possible to have it.

Reply:
This function doesn't even belong here, but I don't care right now. Create a new issue to discuss how to properly implement this please. We still need it merged immediately!

Reply:
This is unacceptable guys. This function shouldn't be here in the first place. Choosing the right HTML file to serve should not be done in C. The web server should be agnostic.